.gitignore

```diff
@@ -2,4 +2,4 @@ Profile
 .DS_Store
 pageshot.xpi
 pageshot.update.rdf
-  
\ No newline at end of file
+*.pyc
```

Comment:
Note you can use `~/.gitconfig` to point to a global gitignore file, and ignore these files forever.

Also I think you might not have your editor configured to always include a trailing newline.